apis/swagger.yml

```diff
@@ -56,6 +56,32 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"
 
+  /auth:
+    post:
+      summary: "Check auth configuration"
+      description: "Validate credentials for a registry and, if available, get an identity token for accessing the registry without password."
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      responses:
+        200:
+          description: "An identity token was generated successfully."
+          schema:
+            $ref: "#/definitions/AuthResponse"
+        204:
```

Comment:
In code implementation, I am afraid I did not find any status code representing 204. In addition, in code implementation, I see you returned `StatusUnauthorized `, and please add this status code 401 in swagger.yml. Please correct me if I missed something.